rpcs3/Crypto/unself.cpp

```diff
@@ -5,10 +5,11 @@
 #include "Emu/FS/vfsLocalFile.h"
 #include "unself.h"
 #pragma warning(push)
-#pragma message("TODO: remove wx dependencies: <wx/mstream.h> <wx/zstream.h>")
+#pragma message("TODO: remove wx dependencies: See comment below.")
 #pragma warning(disable : 4996)
-#include <wx/mstream.h>
-#include <wx/zstream.h>
+
+// TODO: Still reliant on wxWidgets for zlib functions. Alternative solutions?
+#include <wxWidgets/src/zlib/zlib.h>
```

Comment:
Your first suggestion did not build locally, second suggestion did. So I'm trying that one. If this becomes too many commits again, I'll do another squash.